Energy Saver/Pages/Edit.cshtml.cs

```diff
@@ -58,11 +65,13 @@ public async Task<IActionResult> OnPostAsync()
             try
             {
                 await _context.SaveChangesAsync();
+                OnTaxEditSuccess();
             }
             catch (DbUpdateConcurrencyException)
             {
                 if (!TaxesExists(Taxes.ID))
                 {
+                    OnTaxEditError();
                     return NotFound();
```

Comment:
```suggestion
                    return RedirectToPage("./Index");
```

Reply:
There are multiple NotFound() calls, I will change to RedirectToPage()